Safe Browsing V2 is deprecated, switch to V4 by reverting FF56-only fix

diff --git browser/app/profile/firefox.js browser/app/profile/firefox.js
index 276ed7dd30bb..b22d17c999a5 100644
--- browser/app/profile/firefox.js
+++ browser/app/profile/firefox.js
@@ -1709,6 +1709,14 @@
 // Whether or not to restore a session with lazy-browser tabs.
 pref("browser.sessionstore.restore_tabs_lazily", true);
 
+// Enable safebrowsing v4 tables (suffixed by "-proto") update.
+pref("urlclassifier.malwareTable", "goog-malware-proto,goog-unwanted-proto,test-malware-simple,test-unwanted-simple");
+#ifdef MOZILLA_OFFICIAL
+pref("urlclassifier.phishTable", "goog-phish-proto,test-phish-simple");
+#else
+pref("urlclassifier.phishTable", "googpub-phish-proto,test-phish-simple");
+#endif
+
 pref("browser.suppress_first_window_animation", true);
 
 // Preferences for Photon onboarding system extension
diff --git modules/libpref/init/all.js modules/libpref/init/all.js
index 5c0d1a999ff8..d95b0bcfc79e 100644
--- modules/libpref/init/all.js
+++ modules/libpref/init/all.js
@@ -5331,8 +5331,8 @@
 #endif
 
 // Tables for application reputation.
-pref("urlclassifier.downloadAllowTable", "goog-downloadwhite-digest256");
-pref("urlclassifier.downloadBlockTable", "goog-badbinurl-shavar");
+pref("urlclassifier.downloadAllowTable", "goog-downloadwhite-proto");
+pref("urlclassifier.downloadBlockTable", "goog-badbinurl-proto");
 
 pref("urlclassifier.disallow_completions", "test-malware-simple,test-phish-simple,test-unwanted-simple,test-track-simple,test-trackwhite-simple,test-block-simple,goog-downloadwhite-digest256,base-track-digest256,mozstd-trackwhite-digest256,content-track-digest256,mozplugin-block-digest256,mozplugin2-block-digest256,block-flash-digest256,except-flash-digest256,allow-flashallow-digest256,except-flashallow-digest256,block-flashsubdoc-digest256,except-flashsubdoc-digest256,except-flashinfobar-digest256");
 
